Form/Type/ChangePasswordType.php

```diff
@@ -24,13 +24,27 @@ class ChangePasswordType extends AbstractType
      */
     private $security;
 
+    /**
+     * @var int
+     */
+    private $minLength;
+
+    /**
+     * @var int
+     */
+    private $minStrength;
```

Comment:
De kans dat we het ChangePasswordType formulier gaan extenden is best aannemelijk, dan kan je ze beter protected maken zodat we er nog wat mee kunnen doen.